TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/HWC.java

```diff
@@ -15,7 +15,8 @@
  */
 public class HWC {
     // Declare empty variables for robot hardware
-    public DcMotorEx leftFront, rightFront, leftRear, rightRear;
+    public DcMotorEx leftFront, rightFront, leftRear, rightRear; // Wheel motors.
```

Comment:
Separate these into separate declarations, and make the comments Javadoc comments. Or if you feel it's self explanatory just remove the comment all together,

Reply:
Addendum: if you feel it's self explanatory you also don't need to expand the declarations into their own separate lines.